pkg/controllers/resources/pods/syncer.go

```diff
@@ -168,8 +182,47 @@ func (s *podSyncer) ModifyController(registerContext *synccontext.RegisterContex
 			}
 		},
 	}
+	builder.Watches(&corev1.Namespace{}, namespaceEventHandler)
 
-	return builder.Watches(&corev1.Namespace{}, eventHandler), nil
+	// watch for runtime class changes
+	logger := loghelper.New("podSyncer:watcher")
+	loggerInfof := func(format, class, objectName string) {
+		logger.Infof("%s triggers requeue of pods related with %s: %q", format, class, objectName)
+	}
+	runtimeClassEventHandler := handler.Funcs{
+		CreateFunc: func(_ context.Context, e event.CreateEvent, q workqueue.TypedRateLimitingInterface[ctrl.Request]) {
+			loggerInfof("creation", "runtimeClass", e.Object.GetName())
```

Comment:
same comment about logging as above (in pkg/controllers/resources/persistentvolumeclaims/syncer.go)

Reply:
Done!